content/api.md

```diff
@@ -4,31 +4,155 @@ title: API usage
 ---
 ## Introduction
 
-The data collected for the [2FA Directory](https://2fa.directory) website is also available as JSON files to enable developers to use it in their programs. The API with the highest version number is recommended since older versions might not include all available information.
+The data collected for the [2FA Directory][site_url] website is also available as JSON files for developers to use in their programs.
 
-### Caching
+## Supported versions
 
-If you intend to query our JSON files often and with a lot of traffic, you may be blocked by Cloudflare, our reverse proxy provider. We therefore recommend that you cache the files locally for any significant traffic cases.
+The table below provides the End of Life (EOL) dates for different 2FA Directory API versions. Please refer to this table to ensure you are using a supported version.
 
-### Avoid downloading unnecessary data
+|     Version      |          Supported           |
+|:----------------:|:----------------------------:|
+| [Version 4](#v4) |             Yes              |
+| [Version 3](#v3) |             Yes              |
+| [Version 2](#v2) | Unsupported since 2023-08-01 |
+| [Version 1](#v1) | Unsupported since 2023-07-01 |
 
-If you only intend to use a specific dataset, like all sites supporting RFC-6238, we recommend using the URI which lists _just_ that. See [URIs](#uris) for available paths. The smaller, the better.
+## Things to note
+
+When using the 2FA Directory data, there are a few important considerations to keep in mind to ensure compliance and optimal performance.
+
+#### Attribution
+
+The data for the 2FA Directory is licensed under an [MIT license][license] which requires attribution when using the data.
+If the data collected from the 2FA Directory is visible to your users, please provide attribution. This helps acknowledge the source and supports the ongoing maintenance of the directory.  
+At a minimum, please include the following attribution:
+
+> Data sourced from [2FA Directory][site_url] by [2factorauth][repo_url].
```

Comment:
The blockquote doesn't show up on the site.

<img width="1209" alt="Screenshot of the 2FA Directory API page showing the Attribution section" src="https://github.com/user-attachments/assets/70bb98cb-25d0-470a-856d-2424056f3d81">

Reply:
Well, the blockquote is there, but we haven't made any CSS to change the background or changed the text color.

It looks fine to me, but if you prefer some CSS changes, I guess I can also add that.
But maybe we can do it in another PR?

Reply:
Sounds good to me.